integration-tests/opentelemetry-reactive/src/main/java/io/quarkus/it/opentelemetry/reactive/SpanData.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.it.opentelemetry.reactive;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection(classNames = {
+        "io.opentelemetry.sdk.trace.data.SpanData",
```

Comment:
Any reason to not do it in the extension ?

Reply:
This is just for us to retrieve the Span data for assertion over a rest endpoint with the jackson serialization. It is not intended to be used for real application, since telemetry data is pushed to an OTLP exporter.